src/app/zones/views/ZoneEgressListView.vue

```diff
@@ -13,128 +15,94 @@
         },
       ]"
     >
-      <div class="zoneegresses">
-        <div class="stack">
-          <div class="kcard-border">
-            <DataOverview
-              :selected-entity-name="entity?.name"
-              :page-size="PAGE_SIZE_DEFAULT"
-              :is-loading="isLoading"
+      <DataSource
+        v-slot="{ data: zoneEgressOverviewData, error }: ZoneEgressOverviewCollectionSource"
+        :src="`/zones/zone-egresses?size=${props.size}&page=${props.page}`"
+      >
+        <KCard>
+          <template #body>
+            <AppCollection
+              data-testid="zone-egress-table"
+              :headers="HEADERS"
+              :page-number="props.page"
+              :page-size="props.size"
+              :total="zoneEgressOverviewData?.total"
+              :items="zoneEgressOverviewData ? transformToTableData(zoneEgressOverviewData.items) : []"
               :error="error"
-              :empty-state="EMPTY_STATE"
-              :table-data="tableData"
-              :table-data-is-empty="tableData.data.length === 0"
-              :next="nextUrl"
-              :page-offset="pageOffset"
-              @table-action="loadEntity"
-              @load-data="loadData"
-            />
-          </div>
-
-          <div
-            v-if="entity !== null"
-            class="kcard-border"
-            data-testid="list-view-summary"
-          >
-            <ZoneEgressDetails :zone-egress-overview="entity" />
-          </div>
-        </div>
-      </div>
+              @change="route.update"
+            >
+              <template #name="{ row }">
+                <RouterLink
+                  :to="row.detailViewRoute"
```

Comment:
Can we just add this stuff inline?

Reply:
No. I’ve explained several times now that I really don’t want to have all of this in the template and it worked like this before as well. I see no reason to move this into the template.

Reply:
Please can you see if you can inline this, looking at it myself it looks like we can. There's no point reshaping all this data in some function down the bottom of the page when you can just do it inline.

Reply:
I’m not doing this. There is no point in reshaping all this data in the template when we can do so how it is done right now.

Reply:
Same goes for any other places in this PR were we aren't inlining this sort of simple stuff. If its not simple stuff please let me know.